APIAssignment1.py

```diff
@@ -0,0 +1,101 @@
+import json
+import requests
+import csv
+
+# 1. Write a function that would provide the below mentioned rocket and payload details
+# for the latest historical event.
+# Rocket name
+# Rocket height and diameter in meters
+# Payload id
+# Payload nationality and mass in kg
+def rocketdetails():
+    info = [] #stores attributes for output, will write to csv later
+    #set headers
+    info.append(["Rocket_Name", "Rocket_Height", "Rocket_Diameter", "Payload_id", "Payload_nationality", "Payload_mass"])
```

Comment:
If the content of `info` list is static, why do we need to have this extra append operation?